src/lib/annotations/doc/DocDrawingThread.js

```diff
@@ -0,0 +1,92 @@
+import {
+    STATES_DRAW,
+    PAGE_PADDING_TOP,
+    PAGE_PADDING_BOTTOM,
+    CLASS_ANNOTATION_LAYER_DRAW
+} from '../annotationConstants';
+import { getScale } from '../annotatorUtil';
+import DrawingPath from '../drawing/DrawingPath';
+import DrawingThread from '../drawing/DrawingThread';
+import * as DocAnnotatorUtil from './docAnnotatorUtil';
```

Comment:
Will change for consistency, but I would push for classes to be in all caps. Especially since we are loading them in pseudo static contexts.